cdap-distributions/src/packer/files/cdap-sdk.json

```diff
@@ -1,6 +1,6 @@
 {
   "cdap": {
-    "version": "4.1.0-1"
+    "version": "4.1.1"
```

Comment:
This should have the revision, also... `4.1.1-1` as it's a package version (RPM/DEB).